src/Assignment-2/assignment2.html

```diff
@@ -13,6 +13,104 @@
 
 	<!-- Start your code from here -->
 	
+    <div>
+        <header>
+            <div class="firstpart">
+                <div class="headersection">
+                    <img src="https://www.the10xacademy.com/assets/images/logo-color.png" alt="">
+                </div>
+                <div class="headersection">
+                    <div class="headersection-form">
+                        <form>
+                            <input type="text" name="search" placeholder="Search">
+                        </form>
+                    </div>
+                </div>
+            </div>
+            <div class="secondpart">
+                <nav class="navibar">
+                    <ul>
+                        <li><a href="https://google.com">section1</a></li>
```

Comment:
no need to use li tags again for formating in nav bar. try to use only anchor tags inside nav bar